include/rocksdb/db.h

```diff
@@ -621,6 +621,10 @@ class DB {
     // "rocksdb.block-cache-pinned-usage" - returns the memory size for the
     //      entries being pinned.
     static const std::string kBlockCachePinnedUsage;
+
+    // "rocksdb.db_statistics_in_options" - returns multi-line string
+    //      containing the statistics of the db in options
```

Comment:
Maybe saying "options.statistics", rather than "the statistics of the db in the options".